app/src/main/java/eu/kanade/tachiyomi/data/preference/PreferenceKeys.kt

```diff
@@ -147,6 +147,8 @@ object PreferenceKeys {
 
     const val alwaysShowChapterTransition = "always_show_chapter_transition"
 
+    const val showHideScrollBar = "show_hide_scrollBar"
```

Comment:
scollbar is one word so dont camelcase bar

const val showHideScrollbar
show_hide_scrollbar